dbt_subprojects/daily_spellbook/models/_projects/gmx/event/arbitrum/gmx_v2_arbitrum_OrderCreated.sql

```diff
@@ -0,0 +1,270 @@
+{{
+  config(
+    schema = 'gmx_v2_arbitrum',
+    alias = 'order_created',
+    materialized = 'table',
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "gmx",
+                                \'["ai_data_master","gmx-io"]\') }}'
```

Comment:
```suggestion
    materialized = 'table'
```

Reply:
this suggestion assumes you will be eventually building a crosschain spell, which we can then display that final crosschain on data explorer

Reply:
Done